modules/generate/generate.go

```diff
@@ -34,6 +35,64 @@ func GetRandomString(n int) (string, error) {
 	return string(buffer), nil
 }
 
+// GetRandomPassword generates a random password
+func GetRandomPassword(n int) (string, error) {
```

Comment:
I think `GenerateRandomPassword()` or `GeneratePassword()` would be better.

Reply:
golang-lint  send error:
`modules/generate/generate.go:39:6: func name will be used as generate.GeneratePassword by other packages, and that stutters; consider calling this Password (golint)
func GeneratePassword(n int) (string, error) {`
 Renamed to generate.Password()